src/OrchardCore.Modules/OrchardCore.Lucene/Views/LuceneSettings.Edit.cshtml

```diff
@@ -2,17 +2,18 @@
 
 @if (Model.SearchIndexes.Any())
 {
-<div class="form-group" asp-validation-class-for="SearchIndex">
-    <label asp-for="SearchIndex">@T["Default search index"]</label>
-    <select asp-for="SearchIndex" class="form-control">
-        @foreach (var index in Model.SearchIndexes)
-        {
-            <option value="@index" selected="@(Model.SearchIndex == index)">@index</option>
-        }
-    </select>
-    <span asp-validation-for="SearchIndex"></span>
-    <span class="hint">@T["The default index to use for the search page."]</span>
-</div>
+    <div class="form-group" asp-validation-class-for="SearchIndex">
+        <label asp-for="SearchIndex">@T["Default search index"]</label>
+        <select asp-for="SearchIndex" class="form-control">
```

Comment:
Why you didn't use `asp-items` for simplicity & clearity?

Reply:
Probably because `Model.SearchIndexes` is not a `List<SelectListItem>`. Would need to refactor the ViewModel.

Reply:
For now it's Ok, but we could make it one day to make the UI clean